pkg/virt-handler/virtwrap/manager.go

```diff
@@ -416,3 +427,125 @@ func (l *LibvirtDomainManager) KillVM(vm *v1.VM) error {
 	l.recorder.Event(vm, kubev1.EventTypeNormal, v1.Deleted.String(), "VM undefined")
 	return nil
 }
+
```

Comment:
I absolutely do intend to say *never* here. A pluggable virt driver is not something I want to see in here. Any features that VDSM might provide over & above what libvirt provides should be either provided by a layer above KubeVirt, or provided by KubeVirt itself.